src/sentry/api/endpoints/organization_force_auto_assignment.py

```diff
@@ -0,0 +1,47 @@
+from rest_framework.request import Request
+from rest_framework.response import Response
+
+from sentry import ratelimits as ratelimiter
+from sentry.api.base import region_silo_endpoint
+from sentry.api.bases.organization import OrganizationEndpoint
+from sentry.api.serializers import serialize
+from sentry.models import Organization
+from sentry.models.group import Group
+from sentry.models.projectownership import ProjectOwnership
+from sentry.types.ratelimit import RateLimit, RateLimitCategory
+
+
+@region_silo_endpoint
+class OrganizationForceAutoAssignmentEndpoint(OrganizationEndpoint):
```

Comment:
This seems like an endpoint that could be called directly by user. Please make sure to include documentation and publish it. Here's an example: https://github.com/getsentry/sentry/blob/master/src/sentry/api/endpoints/organization_member/details.py#L90

Reply:
We want the endpoint to be directly called by a user, but do not wish to make it public as this can be a labour-intensive call (see [here](https://sentry.slack.com/archives/C04KZQBNQ2U/p1688594072303679?thread_ts=1687793787.448889&cid=C04KZQBNQ2U)). This API will only be called by users who reach out to support and ask for the functionality.

(cc: @NisanthanNanthakumar @brianthi)

Reply:
Thanks for the context. Going towards the goal of API first company we prefer all public APIs to be publicly documented so let's discuss other solutions. Also if it's not public how do they know about it to ask from support?!
1. From the conversation seems like excessive rate limiting is a good idea to address this issue. Does it not?
2. What is the limit? How many orgs can have access to this and why does that limit exists? If you provide more context on the technical challenge maybe we can find a better solution. Hiding it definitely won't work, what if too many orgs find out about it through support to just code?

Reply:
I think this would work. @sentaur-athena what do you think?

For more context, a user would only want to use this API if they manually bulk assigned groups after auto-assignment, then later wanted auto-assignment to run. In general, users can manually assign groups if they made a mistake in assignment. This API was made in response to a customer who had the first use case and reached out to support asking if this was possible. 

Excessive rate limiting could address this issue, but I think it's a rare enough use case that we don't need this labor intensive API to be public.